traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/callbacks/PartnerOfferingFetchCallback.java

```diff
@@ -0,0 +1,10 @@
+package com.guestlogix.travelercorekit.callbacks;
+
+import com.guestlogix.travelercorekit.models.PartnerOfferingGroup;
+
+import java.util.List;
+
+public interface PartnerOfferingFetchCallback {
+    void onSuccess(List<PartnerOfferingGroup> lstPartnerOfferingGroups);
```

Comment:
none of callbacks are documented in core. and i honestly think they dont need because they are not protocole interfaces they are just being used like function pointers and are so self descriptive. 

do you think we should add dcoumentation for all of them or none of them ?